### ISSUE TITLE
Gradle: compose.kotlinCompilerPluginArgs.add("suppressKotlinVersionCompatibilityCheck=1.8.10") only works for JVM targets

### ISSUE THREAD

**Issue by hfhbd** [JetBrains/compose-multiplatform#2734]
**Describe the bug**
I want to suppress the kotlin version check (to use compose-jb with 1.8.10). 
According to the docs, I should use this code:
```
compose {
    kotlinCompilerPlugin.set("1.4.0")
    kotlinCompilerPluginArgs.add("suppressKotlinVersionCompatibilityCheck=1.8.10")
}
```
But this does only suppress the checks in a JVM target, but not in any JS or iOS target. 

**Affected platforms**
Select one of the platforms below:
- Gradle Plugin 

**Versions**
- Kotlin version*: 1.8.10
- Compose Multiplatform version*: 1.4.0
- OS version(s)* (required for Desktop and iOS issues): n/A
- OS architecture (x86 or arm64): n/A

**To Reproduce**
1. Open attatched reproducer
2. Assemble
3. Uncomment the KotlinCompile task options and enable line 47 again to use the documentated configuration
4. clean and build again

**Expected behavior**
A working build suppressing the kotlin version check
[ComposeTodo Kopie.zip](https://github.com/JetBrains/compose-jb/files/10719931/ComposeTodo.Kopie.zip)

**Comment by hfhbd** [comment]
Fixed by #2716

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2734">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
